app/config/config.example.php

```diff
@@ -175,4 +175,8 @@
         'controller' => 'error',
         'action'     => 'index',
     ],
+    'reCaptcha' => [
+        'siteKey' => '6LcLwCcTAAAAAGIDp_P0nRAz7K1DniEmhZ3txWgI',
+        'secret' => '6LcLwCcTAAAAAIW3lwB5NdnQPXVLPzHLmrvcAeC7'
```

Comment:
This is key to test only work on this site http://forum.phanbook.com/post/discussion, so you want to change?